lib/std/fmt/parse_float/parse_float.zig

```diff
@@ -12,6 +12,10 @@ pub const ParseFloatError = error{
 };
 
 pub fn parseFloat(comptime T: type, s: []const u8) ParseFloatError!T {
+    if (T != f16 and T != f32 and T != f64 and T != f128) {
```

Comment:
```suggestion
    if (@typeInfo(T) != .Float) {
```

Reply:
Don't yet support parsing `f80` which this would allow.

Reply:
Shouldn't that be a different compile error, though? `cannot parse a float into a non-floating point type` seems like a confusing compile error if you pass `f80` as the type. Maybe something like this instead?

```
if (@typeInfo(T) != .Float) {
    @compileError("cannot parse a float into a non-floating point type");
}
if (T == f80) {
    @compileError("TODO support parsing float to f80");
}
```

Reply:
Ah, didn't see `@typeInfo`. Learning a lot about Zig with this PR :wink:

Good catch on `f80` - do you usually open issues for TODOs relating to missing features such as this one or do they stay in code?